pennylane/plugins/tests/conftest.py

```diff
@@ -128,9 +131,26 @@ def pytest_addoption(parser):
     """Add command line option to pytest."""
 
     # The options are the three arguments every device takes
-    parser.addoption("--device", action="store", default=None)
-    parser.addoption("--shots", action="store", default=None, type=int)
-    parser.addoption("--analytic", action="store", default=None)
+    parser.addoption("--device", action="store", default=None, help="The device to test.")
+    parser.addoption(
+        "--shots",
+        action="store",
+        default=None,
+        type=int,
+        help="Number of shots to use in stochastic mode.",
+    )
+    parser.addoption(
+        "--analytic",
+        action="store",
+        default=None,
+        help="Whether to run the tests in stochastic or exact mode.",
+    )
+    parser.addoption(
+        "--skip-ops",
+        action="store_true",
+        default=False,

```

Comment:
I'm ambivalent here, I left it like this as that is what @antalszava's original logic did.

An argument for making the skipping optional is given here by @mariaschuld: https://github.com/PennyLaneAI/pennylane/pull/724#pullrequestreview-457535994

Reply:
Thanks! Although I'm not convinced.

If we have a "lenient" and a "strict" mode, which mode do we use to conclude that the device is successfully passing tests? I thought the idea of the test suite was to define a standard, if a device passes the tests then it's good to work with PL.

If passing tests on strict mode is the requirement, then does that mean that there is a canonical set of gates that a device must support? That seems like a high bar, especially on hardware devices.

Reply:
> I thought the idea of the test suite was to define a standard, if a device passes the tests then it's good to work with PL.

I think the idea is more to (a) reduce device integration test duplication, and (b) reduce maintenance load. Using it to set a standard is a potential option we marked to explore further down the road :)

Reply:
Agree with the points raised by Josh. From my point of view, if a device for some reason cannot support all operations then basically it also cannot use the shared test suite. This would simply lead to having to duplicate the tests in another repo.